docs/user_guide/hybrid_queries_02.ipynb

```diff
@@ -95,8 +95,8 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "\u001b[32m19:22:36\u001b[0m \u001b[35msam.partee-NW9MQX5Y74\u001b[0m \u001b[34mredisvl.cli.index[21909]\u001b[0m \u001b[1;30mINFO\u001b[0m Indices:\n",
-      "\u001b[32m19:22:36\u001b[0m \u001b[35msam.partee-NW9MQX5Y74\u001b[0m \u001b[34mredisvl.cli.index[21909]\u001b[0m \u001b[1;30mINFO\u001b[0m 1. user_index\n"
+      "\u001b[32m18:11:16\u001b[0m \u001b[34m[RedisVL]\u001b[0m \u001b[1;30mINFO\u001b[0m   Indices:\n",
```

Comment:
Love the new notebook for hybrid search. When we merge, I would just wholesale replace the existing notebook if possible. Might need to manually do that locally. Merge conflicts might be tough here